docs/rst/user_manual/wan_configuration.rst

```diff
@@ -243,17 +234,12 @@ User *B* should set its *listening-addresses* and *connection-addresses* as foll
     - name: WANClientParticipant_userB
       kind: wan
 
-      discovery-server-guid:
-        id: 3                             # Must be different than A one
       listening-addresses:
         - ip: 2.2.2.2                     # Public IP of host Hb
           port: 11777                     # Internal and External port
```

Comment:
(In 3.5.3 Network Address) Say between parenthesis in external port description that it only applies to TCP.